adding MDN review guide (#175)
mdn/content: REVIEWING.md

* "Drive-by fixes" — small updates done to the site to fix typos, grammatical issues, and technical inaccuracies, usually as they are found by users of MDN.
* MDN content bug fixes, usually done by volunteers to close issues opened by users on our [sprints](https://github.com/mdn/sprints/issues) repo.

Regardless of how a content change is done, they will be submitted as [pull requests](https://github.com/mdn/content/pulls) on this repo, which will require rapid reviewing and merging to ensure that the site does not get out-of-date. This is being handled as follows:

1. Different MDN staff members and volunteers have been assigned as "topic review owners", meaning that when a pull request comes in related to a particular topic area of the site (e.g. the CSS reference, or the learning area), it will be assigned to that area's topic review owner(s) and they will receive an e-mail notification asking for a review. This is being handled using a [CODEOWNERS](https://github.com/mdn/content/blob/main/.github/CODEOWNERS) file, in which particular content directories are assigned to the topics review owner's GitHub usernames.

> 1. Once the review has been done and the pull request has been approved, our mdn-merge group will be automatically pinged to ask if one of them can merge the pull request.

Reviewer: The reviewer should manually merge the pull request upon approval. Maybe something like:
```suggestion
1. Once the review has been done and the pull request has been approved, the reviewer should also merge the pull request.
```

Author: Agreed, will make this change

1. Familiarize yourself with the [MDN Code example guidelines](https://developer.mozilla.org/en-US/docs/MDN/Guidelines/Code_guidelines) and make sure that code examples follow the guidelines. You'll get used to them eventually, and we are intending to automatically lint against our guidelines at some point in the future.
1. Familiarize yourself with the [MDN Writing style guide](https://developer.mozilla.org/en-US/docs/MDN/Guidelines/Writing_style_guide), and use it to inform your reviews of new text content.
1. Familiarize yourself with the MDN [pull request guidelines](https://github.com/mdn/content/blob/main/README.md#pull-request-etiquette). The key points here are
   * You have the right to request more information to help your review if the submitter has not explained why they are making this change.
   * You have the right to close a pull request if it is too complex and/or contains multiple unrelated changes and ask the submitter to submit their changes in smaller atomic chunks.

> 1. When reviewing a pull request, use the [GitHub review tools](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews). Use "Request changes" when submitting a review that will require the submitter to do some more work, or "Approve" if the submission is ready to add and you want to request merging. [Reviewing proposed changes in a pull request](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/reviewing-proposed-changes-in-a-pull-request) is also useful if you want more information.

Reviewer: ```suggestion
1. When reviewing a pull request, use the [GitHub review tools](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews). Use "Request changes" when submitting a review that will require the submitter to do some more work, or "Approve" if the submission is ready to add and you want to merge it. [Reviewing proposed changes in a pull request](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/reviewing-proposed-changes-in-a-pull-request) is also useful if you want more information.
```

Author: Agreed, will change.

1. Be polite and constructive at all times when writing review comments, or otherwise interacting with the submitter. You and the submitter are both bound by [our Code of Conduct](CODE_OF_CONDUCT.md) when contributing to MDN, which means adhering to Mozilla's [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/). If anyone has engaged in behavior that is potentially illegal or makes you or someone else feel unsafe, unwelcome, or uncomfortable, you are encouraged to [report it](https://www.mozilla.org/en-US/about/governance/policies/participation/reporting/). We want MDN to be a welcoming, friendly community that we can all be proud of.
1. If you don't understand a content change that you've been selected to review, or feel that it is too large and complex for you to deal with, don't panic! Feel free to reach out to someone else to ask for help, like a colleague, or someone else in your group of topic review owners (if you know who they are). If you are not sure who to approach for help, then ping our @core-yari-content group to ask for help.

> 1. Related to the above point, it is rare that you'll be required to review a large, complex content change with no warning, like a complete page rewrite, or the addition of several new reference pages or tutorials. Usually such changes are done as part of specific work streams where the content has been approved for addition, and reviewer(s) have been assigned already. In such cases, the PR should be linked to an issue that explains all these details. If you are not sure, ask the submitter if they need a review of the content, and/or if their submission has been approved for inclusion on MDN. Ping our @core-yari-content group to ask for help if you are still not sure, or if you think the content is suspicious.

Reviewer 2: >ask the submitter if ... their submission has been approved for inclusion on MDN.

This sounds like it was written non-reference content. If a feature comes out of a spec committee and a browser implements a stable version, getting approval seems superfluous.

In any case, I'd still expect there to be an issue to announce, at the very least, that something is being worked on so that we don't have duplicate efforts.

Author: I agree; I've tweaked this a bit.

 

## Topic review owners

The following specific topic areas are being reviewed by the kind souls listed underneath them. Be kind to them, and thank them for all the help they give to this project. If you would like to help with MDN content reviews, [get in touch with us](https://developer.mozilla.org/en-US/docs/MDN/Contribute/Getting_started#Step_4_Ask_for_help).

Note that changes to any content areas not explicitly listed below will be handled by the @core-yari-content group, which currently consists of @chrisdavidmills.

> 

Reviewer 3: Code owners is now team based (https://github.com/mdn/content/pull/177). Should we/can we link to the teams themselves as well? [not "instead" - it is good we mention the people here]

Author: that is a good idea. I'll add the teams too.
